.changes/next-release/Bug Fix-de534112-c087-411d-93e0-e38a7d76e10b.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "CodeWhisperer: Fix a bug when CodeWhisperer sometimes add extra space at the beginning of suggestion"
```

"bug fix" is already the item prefix so it is never necessary to repeat that in the description
```suggestion
	"description": "CodeWhisperer sometimes add extra space at the beginning of suggestion"
```